extensions/vscode/package.json

```diff
@@ -92,6 +96,13 @@
           "group": "dartFrogGroup@2",
           "when": "!(resourceDirname =~ /routes/) && !(explorerResourceIsFolder && resourceFilename == routes)"
         }
+      ],
+      "debug/toolBar": [
+        {
+          "command": "dart-frog.stop-dev-server",
+          "group": "navigation@50",
```

Comment:
This label is quite important, and somewhat tricky to find what the correct order (50 in this case) should be used. I had to do some diving and information can be seen [here](https://github.com/microsoft/vscode/blob/05f926991c9c52dff838d542c0014acd756833e2/src/vs/workbench/contrib/debug/electron-browser/debug.contribution.ts#L262). There is a tracking issue asking for documentation [here](https://github.com/microsoft/vscode/issues/192185).